west.yml

```diff
@@ -137,19 +137,19 @@ manifest:
     - name: mbedtls
       path: modules/crypto/mbedtls
       repo-path: sdk-mbedtls
-      revision: b57377329f73f17ebb7d6e95e1da7f2e5a40f57a
+      revision: v3.3.0-ncs1-rc1
```

Comment:
The ncsx part should be bumped, given it's the second release on this mbed TLS version, so `v3.3.0-ncs2-rc1`

Reply:
I can do that